app/mailers/customer_mailer.rb

```diff
@@ -37,10 +37,13 @@ def receipt(purchase_id = nil, charge_id = nil, for_email: true)
 
     is_receipt_for_gift_receiver = receipt_for_gift_receiver?(@chargeable)
     @footer_template = "layouts/mailers/receipt_footer" unless is_receipt_for_gift_receiver
+    purchase = @chargeable.is_a?(Purchase) ? @chargeable : @chargeable.purchases.last
+    reply_to = purchase&.link&.reply_to_email&.email || @chargeable.seller.support_or_form_email
```

Comment:
May be better to call this support_email and then it's clear / doesn't need to be changed as much. product.support_email can fallback to @chargeable.seller.support_or_form_email etc from the product spec itself